test/runtime/translator/translator.translate.selector.test.js

```diff
@@ -0,0 +1,56 @@
+import { describe, it, expect, beforeAll } from 'vitest';
+import Translator from '../../../src/Translator.js';
+import ResourceStore from '../../../src/ResourceStore.js';
+import LanguageUtils from '../../../src/LanguageUtils.js';
+import PluralResolver from '../../../src/PluralResolver.js';
+import Interpolator from '../../../src/Interpolator.js';
+
+describe('Translator', () => {
+  describe('translate() with selector and custom separators', () => {
+    /** @type {Translator} */
+    let t;
+
+    beforeAll(() => {
+      const rs = new ResourceStore({
+        en: {
+          root: {
+            foo: {
+              bar: 'foobar'
+            }
+          },
+        },
+      });
```

Comment:
This test is wrong... sorry.
The keySeparator options etc... needs also to be passed to the resource store...
Will close this PR... a fix is coming...